### ISSUE TITLE
Is it possible to have multiple node-dota2 instance on with different accounts?

### ISSUE THREAD

**Issue by gerlaic** [Arcana/node-dota2#342]
I want to use multi processes to have 4 node-dota2 on at the same time. Is this possible?

**Comment by jimmydorry** [comment]
Sure. The easiest way would be to copy the node-dota2 files, and setup each normally. If you want to be doing something more complex, you are going to need to have a read of the files and the way the library works. Certainly possible, but not out of the box.

Have a look at the examples. Looks straight forward enough.

**Comment by Crazy-Duck** [comment]
Shameless plug, but you might wanna take a look at [this](https://github.com/Crazy-Duck/node-dota2-bot). It's a wrapper I've made around the library, which packages the entire thing in a bot object and handles the login, queueing, etc. If you'd want to run multiple instances in parallel, you'd just need to instantiate a series of bot objects. As soon as I've gotten time to figure out how to do it, I'll probably try to bring it under the Arcana org as well.

**Comment by jimmydorry** [comment]
I seriously can't wait. While node-dota2 gives all the power you want, it's a bit daunting to start with... especially for people new to nodejs.

**Comment by Crazy-Duck** [comment]
Seems like I have insufficient rights to transfer the repo to Arcana :/ Apparently you need admin rights on the organization before you can transfer ownership to it -_-'

**Comment by jimmydorry** [comment]
Haha, you're right. I just checked. Let's take this to another issue.
